challenge_1/helloworld/src/test/scala/example/HelloSpec.scala

```diff
@@ -0,0 +1,10 @@
+package example
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class HelloSpec extends AnyFlatSpec with Matchers {
+  "The Hello object" should "say hello" in {
+    Hello.greeting shouldEqual "hello"
```

Comment:
if u try to run "sbt test" on the command line , it would fail because the output is now "30Days of Scala" 
```suggestion
    Hello.greeting shouldEqual "Welcome to 30 days of Scala"
```

Reply:
Yes I noticed this while attempting to solve the second task. I have made corrections to the code.